### DISCUSSION TITLE
Download

### DISCUSSION THREAD

**Discussion by paolomaccallini-hub** [broadinstitute/seqr#4422] [Q&A] [answered]
Why is it not always possible to download the results of a search?
It seems to be linked to the number of variants included in the result. 
Is it intended to be this way?

**Comment by hanars** [comment, marked as answer]
sear does not allow downloading more than 1000 variants as this is not what seqr is designed to do. Building an application that allows you to quickly query a human-analysable list of a couple hundered variants is a totally different engineering problem than building a backend that allows you to export thousands of variants. seqr is not a workflow engine - it is designed to be used to allow analysts to do deep dives into targeted variants in the application. If your goal is to annotate and then export thousands of variants, you should not be using seqr
